src/routes/api/unsplash.ts

```diff
@@ -25,7 +35,7 @@ export async function get(event: RequestEvent) {
   });
 
   const result = await unsplash.search.getPhotos({
-    query: 'beach'
+    query: 'mountain'
   });
```

Comment:
I believe this will load the same set of pictures, but we can get back to this later.